### PR TITLE
Cloud Backups Upgrade

### PR DESCRIPTION
**Pull request by brunobar79**
- Force cloud sync when launching the app (iOS)
- Better handling of syncing issues
- Add the option to import multiple backups  from settings => backup => manage
- Android improvements included in new version of `react-native-cloud-fs`
- Hide backup option if there's only one readonly wallet

Fixes  RNBW-1083

### CODE REVIEW

**Comment by brunobar79** [comment]
Fixes RNBW-1083

**Review by mikedemarais — APPROVED** [review]

File: src/components/settings-menu/SettingsSection.js [hunk 1]
```diff
@@ -114,7 +114,7 @@ const checkAllWallets = wallets => {
     ) {
       areBackedUp = false;
     }
-    if (!wallets[key].type !== WalletTypes.readOnly) {
```
Comment (jinchung) [inline src/components/settings-menu/SettingsSection.js:117]: wtf

File: src/components/backup/RestoreCloudStep.js [hunk 2]
```diff
@@ -128,15 +144,60 @@ export default function RestoreCloudStep({ userData }) {
...
         goBack();
+
         InteractionManager.runAfterInteractions(async () => {
-          replace(Routes.SWIPE_LAYOUT);
+          const wallets = await dispatch(walletsLoadState());
+          if (!userData && backupSelected?.name) {
+            goBack();
+            logger.log('updating redux state of new wallets');
+            // Reload the redux store
+            logger.log('done', wallets);
```
Comment (jinchung) [inline src/components/backup/RestoreCloudStep.js:170]: logging on this line and above are a bit confusing

**Review by skylarbarrera — APPROVED** [review]
🤝

**Review by jinchung — APPROVED** [review]
🍟

**Pull request merged**